src/util.ts

```diff
@@ -60,6 +64,18 @@ export function getMinikubeHome(): string | undefined {
   }
 }
 
+export function getKubeConfig(): string | undefined {
+  const kubernetesConfiguration = extensionApi.configuration.getConfiguration('kubernetes');
```

Comment:
i don't really like this, due to it's a reference to plugin configuration, which may be turned off (and also injects a new implicit dependency on configuration naming), but it looks like there is no other way :c

Reply:
This is okey